### PR TITLE
JBPM-6674: Stunner - Add BPMN Intermediate Message Throwing Event

### PR DESCRIPTION
**Pull request by alepintus**
Hi @romartin @hasys  @wmedvede ,
Intermediate Message Throwing Event added.

Could you take a look please?

Thanks!

### CODE REVIEW

**Comment by romartin** [comment]
ok to test

**Comment by romartin** [comment]
Jenkins execute full downstream build

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/main/resources/org/kie/workbench/common/stunner/bpmn/client/resources/images/icons/event/event-intermediate.svg [hunk 1]
```diff
@@ -26,10 +26,7 @@
 	<g transform="translate(3.7,3.7) scale(0.05,0.05)" style="opacity:1">
 		<use xlink:href="event-signal.svg#eventSignal"/>
 	</g>
-        <g stunner:layout="CENTER" transform="translate(4,4) scale(0.05,0.05)" style="opacity:1">
-    	  <use xlink:href="event-message.svg#eventMessage"/>
-        </g>
-	<g style="opacity:1">
```
Comment (romartin) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/main/resources/org/kie/workbench/common/stunner/bpmn/client/resources/images/icons/event/event-intermediate.svg]: Hey  @alepintus I think here there is an error, it seems you removed the group for `eventIntermediateError` as well, but it should not be removed. Well testing it I see that the icon for the intermediate error message is wrong, it's missing the central part.

**Comment by alepintus** [comment]
Hi @romartin ,
event-intermediate.svg fixed.

Thanks!

**Pull request merged**